rules/no-array-reduce.js

```diff
@@ -4,7 +4,7 @@ const {isNodeValueNotFunction, isArrayPrototypeProperty} = require('./utils/inde
 
 const MESSAGE_ID = 'no-reduce';
 const messages = {
-	[MESSAGE_ID]: '`Array#{{method}}()` is not allowed',
+	[MESSAGE_ID]: '`Array#{{method}}()` is not allowed. Prefer other types of loop for readability.',
```

Comment:
I’ll alter this message to specifically mention .toReversed() for reduceRight. https://github.com/sindresorhus/eslint-plugin-unicorn/pull/2428 isn’t out yet so it’s possible that they will use `.reverse()` instead.